src/app/modules/learning/knowledge-component/learning-objects/instructional-items/video/video.component.ts

```diff
@@ -1,16 +1,30 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import { LearningObjectComponent } from '../../learning-object-component';
 import { Video } from './video.model';
+import {VideoService} from "./video.service";
 
 @Component({
   selector: 'cc-video',
   templateUrl: './video.component.html',
   styleUrls: ['./video.component.css']
 })
-export class VideoComponent implements LearningObjectComponent {
+export class VideoComponent implements LearningObjectComponent, OnInit {
 
   learningObject: Video;
 
-  constructor() { }
+  constructor(private service: VideoService) { }
 
+  ngOnInit(): void {
+    this.service.updateVideoStatus();
```

Comment:
Curiosity question: Is this used to reset the VideoStatus because the service in singleton and components are not?
If so, real smart!

Reply:
Start could indicate that video was started, maybe reset?

Reply:
Maybe, but also consider that the code will be `videoPlaybackMonitor.start()` so it is not too ambiguous and aligns with the convention used in `sessionPauseTracker.start()`.